src/serialization/sb2.js

```diff
@@ -19,6 +19,21 @@ const {loadCostume} = require('../import/load-costume.js');
 const {loadSound} = require('../import/load-sound.js');
 const {deserializeCostume, deserializeSound} = require('./deserialize-assets.js');
 
+// Constants used during deserialization of an SB3 file
+const CORE_EXTENSIONS = [
```

Comment:
Ideally this would live outside of serialization and look at the actual "core" opcodes, but that would require inspecting the runtime and exposing `defaultBlockPackages` which would be fairly invasive.